src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -52,8 +52,8 @@ JAVACPP_BRANCH=${JAVACPP_BRANCH:="https://github.com/bytedeco/javacpp-presets.gi
 JAVACPP_BRANCH_TAG=${JAVACPP_BRANCH_TAG:="master"}
 CMAKE_VERSION=${CMAKE_VERSION:="3.21.1"}
 export JAR_INSTALL_PATH="/workspace/install/java-api-bindings"
-# Note: Following bashscript convention where True/success == 0 and False == 1
-export INCLUDE_DEVELOPER_TOOLS_SERVER=1
+# Note: True/success == 1 and False != 0
```

Comment:
I think this should be True/success != 0 and False == 0.

Reply:
Also, just want to confirm: no documentation or tests need to be updated or will be impacted by this?

Reply:
Wait, I am getting quite confused. I thought this is what we do everywhere, even here (you remove it only if False == 0):
`if [ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -eq 0 ]; then
    rm -r tritonserver/src/gen
    rm tritonserver/src/main/java/org/bytedeco/tritonserver/presets/tritondevelopertoolsserver.java
fi`

Reply:
For example in  [L0_infer](https://github.com/triton-inference-server/server/blob/main/qa/L0_infer/test.sh#L60) we check if variables are equal to 1, because we don't want to test extra things.

Reply:
Thanks for explaining! Interesting. At some point, it would be good to standardize everything.

Thanks for making these changes!